kiwi/tasks/system_prepare.py

```diff
@@ -207,6 +220,7 @@ def process(self):                                      # noqa: C901
         # make sure manager instance is cleaned up now
         del manager
 
+        log.info('Preparing repos')
```

Comment:
imho should be more verbose, preparing what repos ?

Reply:
:sweat:  Damn, should not be here, this from days ago while debugging an image...